StepsToCare/CareEntry.ascx.cs

```diff
@@ -918,8 +1171,13 @@ private void LoadDropDowns( CareNeed careNeed )
         /// <summary>
         /// Binds the Assigned Persons grid.
         /// </summary>
-        private void BindAssignedPersonsGrid()
+        private void BindAssignedPersonsGrid( bool initialLoad = false )
         {
+            if ( !initialLoad )
+            {
+                GenerateTempNeed( null );
```

Comment:
Is this working as expected? Category selection is no longer reloading attributes with qualifiers.